taskvine/src/worker/vine_worker.c

```diff
@@ -198,6 +198,10 @@ static struct itable *procs_complete = NULL;
 // Table of current transfers and their id
 static struct hash_table *current_transfers = NULL;
 
+// Table of all library processes.
```

Comment:
Suggest that we not introduce a separate table for libraries, since that is one more thing to worry about cleaning up properly.  The total number of processes is small, so it seems sufficient to just iterate over procs_running and act on those that are libraries.